app/assets/javascripts/directives/views/accountMenu.js

```diff
@@ -377,6 +377,17 @@ class AccountMenuCtrl extends PureCtrl {
   }
 
   async performImport(data, password) {
+    if (data.keyParams) {
+      if (Number(data.keyParams.version) > 3) {
```

Comment:
You should be able to use this function instead: https://github.com/standardnotes/snjs/blob/master/lib/protocol/manager.js#L25